src/components/atoms/toaster/toaster.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react"
+import React from "react"
+import Toaster from "."
+
+export default {
+  title: "Atoms/Toaster",
+  component: Toaster.Content,
+} as ComponentMeta<typeof Toaster.Content>
+
+const Template: ComponentStory<typeof Toaster.Content> = (args) => (
+  <div className="flex items-start bg-grey-0 p-base border border-grey-20 rounded-rounded shadow-toaster w-[380px]">
+    <Toaster.Content {...args} />
+  </div>
```

Comment:
nit: not sure if this is intended, but this is how the stories look like:
![image](https://user-images.githubusercontent.com/33696020/153413821-d435ccdd-047f-4639-8157-d9de001b56d7.png)